### DISCUSSION TITLE
Trivy-Operator Unable to Use Offline Vulnerability Database in MinIO Bucket

### DISCUSSION THREAD

**Discussion by willie-lin** [aquasecurity/trivy-operator#1955] [Help & Support]
Content: I am trying to use Trivy-Operator in a Kubernetes cluster that does not have internet access. I have stored the offline vulnerability database in a MinIO bucket within the cluster, but I cannot find a way to configure Trivy-Operator to use this offline database instead of updating from the internet.

I have tried changing the value of trivy.dbRepository to the path of the database file in my MinIO bucket, and I have created a Secret to store the credentials needed to access my MinIO bucket. However, it seems that Trivy-Operator is still trying to read the database from a Docker image instead of from my MinIO bucket.

I would like to know if there is a way for Trivy-Operator to use the offline vulnerability database I provided, instead of trying to update from the internet. If there is, how should I configure Trivy-Operator? If not, are there any plans to add this feature?

Thank you!

**Comment by chen-keinan** [comment]
@willie-lin have you consider using private registry, local one ?
